wgpu-core/src/command/render.rs

```diff
@@ -1447,7 +1458,7 @@ impl Global {
             Err(e) => return make_err(e, arc_desc),
         };
 
-        let err = fill_arc_desc(hub, desc, &mut arc_desc).err();
+        let err = fill_arc_desc(hub, desc, &mut arc_desc, &cmd_buf.device).err();
```

Comment:
Should there also be `snatchable_lock.read()`?

Reply:
The snatch lock is only needed for the `Snatchable` fields of a few types.